Lib/zoneinfo/_common.py

```diff
@@ -14,13 +14,23 @@ def load_tzdata(key):
         if path.is_dir():
             raise IsADirectoryError
         return path.open("rb")
-    except (ImportError, FileNotFoundError, UnicodeEncodeError, IsADirectoryError):
-        # There are four types of exception that can be raised that all amount
-        # to "we cannot find this key":
+    except ImportError:
+        # If package_name doesn't exist, it means tzdata is not installed
+        # or there's an error in the folder name like Amrica/New_York
+        msg = (
+            f"No time zone found with key {key}.\n\n"
+            "This error may occur if timezone data is not available. "
+            "To resolve this:\n"
+            "  • Install the tzdata package: python -m pip install tzdata\n"
```

Comment:
```suggestion
            "  - Install the tzdata package: python -m pip install tzdata\n"
```
It is better to not use unicode characters, they can cause problems on systems with non-utf8 file system encodings.

Reply:
Replaced the unicode bullet points with hyphens.